dev-db/pgadmin4/pgadmin4-4.21.ebuild

```diff
@@ -0,0 +1,120 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7,8} )
+PYTHON_REQ_USE="sqlite"
+inherit desktop python-single-r1 qmake-utils xdg
+
+DESCRIPTION="GUI administration and development platform for PostgreSQL"
+HOMEPAGE="https://www.pgadmin.org/"
+SRC_URI="https://ftp.postgresql.org/pub/pgadmin/${PN}/v${PV}/source/${P}.tar.gz"
+
+LICENSE="POSTGRESQL"
+SLOT="0"
+KEYWORDS="amd64 x86"
```

Comment:
Don't commit straight to stable.

Reply:
By the way, repoman would have prevented this.

Reply:
Thanks a lot for pointing it out! I will definitelly try moving to `repoman commit` functionality instead of plain `git`. Fixed now.